runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -394,8 +395,10 @@ else if (groundItemTimers == DespawnTimerMode.SECONDS || groundItemTimers == Des
 
 	private Instant calculateDespawnTime(GroundItem groundItem)
 	{
-		// We can only accurately guess despawn times for our own pvm loot and dropped items
-		if (groundItem.getLootType() != LootType.PVM && groundItem.getLootType() != LootType.DROPPED)
+		// We can only accurately guess despawn times for our own pvm loot, dropped items
+		// or for items placed on tables
+		if (groundItem.getLootType() != LootType.PVM && groundItem.getLootType() != LootType.DROPPED
+			&& groundItem.getLootType() != LootType.TABLE)
```

Comment:
Let's make this one type per line now that we're breaking to another line to begin with.
```suggestion
		if (groundItem.getLootType() != LootType.PVM
			&& groundItem.getLootType() != LootType.DROPPED
			&& groundItem.getLootType() != LootType.TABLE)
```